recipes/toolchain/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: toolchain
```

Comment:
how about prefixing this with `cf`, like `cf-toolchain` since it's conda-forge related